### DISCUSSION TITLE
India Financial Markets Data in FinGPT

### DISCUSSION THREAD

**Discussion by sumanpapanaboina1983** [AI4Finance-Foundation/FinGPT#55] [General]
Hello,
First of all thank you for creating such a wonderful opensource project and community. Would like to know is there any plans to make India Financial Markets Data and Pretrained model as part of FinGPT. Is there are any challenges you are seeing in bringing this data ? or it is already in your roadmap ?

**Comment by YangletLiu** [comment]
> Hello,
> First of all thank you for creating such a wonderful opensource project and community. Would like to know is there any plans to make India Financial Markets Data and Pretrained model as part of FinGPT. Is there are any challenges you are seeing in bringing this data ? or it is already in your roadmap ?

Your suggestion is great!!  Would you like to play with Idian market and share your experiences with our community?

**Reply by psuman** [reply]
Sure, i am more than happy to play with Indian market data, Need some guidance initially to make sure I am not going in wrong direction, what is the best way to get this guidance, just discuss through GitHub discussions by posting the questions ?

**Comment by nishantsolanki89** [comment]
Also I am interested into experiment for indian markets. I tried to train model but wasn't sure on steps so would be helpful if any guidance around that
